openapi-cli/src/test/resources/generators/service/ballerina/response/scenario_04_rs.bal

```diff
@@ -3,6 +3,6 @@ import ballerina/http;
 listener http:Listener ep0 = new (80, config = {host: "petstore.openapi.io"});
 
 service /v1 on ep0 {
-    resource function post user() returns http:Ok|http:BadRequest|http:Unauthorized|http:NotFound {
+    resource function post user() returns http:BadRequest|http:Unauthorized|http:Ok|http:NotFound {
```

Comment:
Yeah, this happened because the implementation was changed which used to generate union type.

Reply:
Ack.So can you explain the new logic which determines the order?

Reply:
In the previous function implementation, It needed to add support to handle multiple response types and common types. Therefore adding those to existing functions may cause to some cost of iteration and errors. Therefore, Here used sets instead of list to avoid duplication. 
The reason for getting changed order, the set interface does not provide any ordering guarantees. Here I have used HashSet. I will change this to the LinkedHashSet to manage the order.